### DISCUSSION TITLE
"float" object has no attribute 'numerator'?

### DISCUSSION THREAD

**Discussion by Maleku611** [nagadomi/nunif#240] [General]
Receiving this error on about 40% of videos I attempt.  Any idea on what might be causing this, I am using MP4 video files.[
<img width="809" alt="Screenshot 2024-10-10 183001" src="https://github.com/user-attachments/assets/be57c453-c00e-42e2-aff5-c9653f333a75">
](url)

**Comment by nagadomi** [comment]
Recently released pyav 13.1.0 is the cause of this problem.
I fixed it to use pyav 13.0.0 for the moment.
Run `update.bat` and try again.

**Reply by Maleku611** [reply]
Okay thanks.  I ran the update and that error stopped, however now receiving this new error.
<img width="813" alt="Screenshot 2024-10-10 192425" src="https://github.com/user-attachments/assets/e2b97132-4846-41ba-9eed-d5f1067c086f">

**Reply by Maleku611** [reply]
Please disregard, realized I have the Depth Batch too high... works great now, thanks!

**Reply by nagadomi** [reply]
Yes, pytorch sometimes give random runtime errors instead of `CUDA out of memory`.
